src/javascript/app/common/currency.js

```diff
@@ -41,7 +41,9 @@ const addComma = (num, decimal_points, is_crypto) => {
     ));
 };
 
-const getFiatDecimalPlaces = () => jpClient() ? 0 : 2;
+const isJPClient = () => (LocalStore.getObject('client.accounts')[LocalStore.get('active_loginid')] || {}).is_jp;
```

Comment:
can we use `Client.get('is_jp')` here?

Reply:
no, because client is calling currency.js, so currency.js can't call client or it will create a circular dependency. we can fix this when we change to import instead of require